### DISCUSSION TITLE
Team Reborn's Energy.

### DISCUSSION THREAD

**Discussion by zenfyrdev** [Sinytra/Connector#75] [Ideas]
On Fabric, most mods use Team Reborn's [Energy](https://github.com/TechReborn/Energy) API, making them incompatible with Forge tech mods.

Energy is identical to the Fabric Transfer API.
But I'm not really the one to discuss implementation with, as I'm not familiar with Forge's energy conventions.

Since Connector is in beta, this isn't a priority, but something that may be supported in the future.

**Comment by Abbie5** [comment]
if tr energy and forge energy are bridged a config option should be added to disable this behaviour, imo

**Comment by Su5eD** [comment]
This is a good idea, but I'd like to keep it separate from Connector itself. Something like "Connector Extras" could fit, which would provide integration with 3rd party APIs, such as modmenu, TR energy, JEI/REI/EMI bridges etc.
